src/core.ts

```diff
@@ -26,7 +28,11 @@ export const commandsConfiguration: {
     gitlab: { job: { tags: ["linux-docker"], variables: {} } },
     commandStart: ['"$PIPELINE_SCRIPTS_DIR/try-runtime-bot.sh"'],
   },
-  fmt: { gitlab: { job: { tags: ["linux-docker"], variables: {} } }, commandStart: ['"$PIPELINE_SCRIPTS_DIR/fmt.sh"'] },
+  fmt: {
+    gitlab: { job: { tags: ["linux-docker"], variables: {} } },
+    commandStart: ['"$PIPELINE_SCRIPTS_DIR/fmt.sh"'],
+    optionalCommandArgs: true,
+  },
```

Comment:
This configuration object is just one more evidence that this `command-bot` / `pipeline-scripts` division didn't bring us any decoupling.